profile/readme.md

```diff
@@ -17,3 +17,11 @@ We also openly licence our code where it is possible to do so.
 ### Oak Web Application
 
 Our main public-facing application is the open source [Oak Web Application](https://github.com/oaknational/Oak-Web-Application), built in [Next.js](https://nextjs.org/), [React](https://react.dev/), [Styled-Components](https://styled-components.com/) and using the [Oak React components library](https://github.com/oaknational/oak-components).
+
+## Our approach to our technical architecture
+
+We take a serverless-first approach to hosting, with the majority of our applications and services running on dedicated Platform-as-a-Service (PaaS) and Infrastructure-as-a-Service (IaaS) provider platforms. 
+
+Our web applications are built and hosted on [Netlify](https://www.netlify.com/), with a handful on [Vercel](https://vercel.com/), while many of our supporting services are either outsourced or run on [Google Cloud Functions](https://cloud.google.com/functions/?hl=en). 
+
+We are in the process of moving all of our infrastructure management to [Terraform](https://github.com/oaknational/Cloud-Config). 
```

Comment:
Right now, Cloud Config (https://github.com/oaknational/Cloud-Config) is private. Not sure if we wanna omit the link? 

Part of the work I've been doing works to make this project potentially public, I could add this to a new Task in Notion to do that as part of that Epic?

Reply:
I'd omit the link whilst it's private. I'm happy for this to be public, providing we're comfortable with all the usual security caveats (i.e no risk of legacy secrets in there etc)